contracts/ConnectivityTrackerHbbft.sol

```diff
@@ -169,6 +171,19 @@ contract ConnectivityTrackerHbbft is Initializable, OwnableUpgradeable, IConnect
 
         reportDisallowPeriod = _reportDisallowPeriodSeconds;
         earlyEpochEndToleranceLevel = 2;
+        
+        uint256 step = 3 * 60; // 3 minutes in seconds
+        uint256[] memory reportDisallowPeriodAllowedParams = new uint256[](10);
+
+        for (uint256 i = 0; i < 10; i++) {
+            reportDisallowPeriodAllowedParams[i] = step + (i * step);
+        }
+
+        __initAllowedChangeableParameter(
```

Comment:
Please provide also a `reinitializer` function to set allowed changeable parameters, because initializer will not be called during upgrade, **[reference](https://docs.openzeppelin.com/contracts/5.x/api/proxy#Initializable-reinitializer-uint64-)**